src/styles/_variables.scss

```
@@ -0,0 +1,13 @@
+$color-lavender: #e1d3e2;
+$color-yellow: #f7cc65;
+$color-teal: #57a48d;
+$color-blue: #345fa2;
+$color-orange: #f27554;
+$color-pink: #dd648d;
+
+$color-ink: #0a0e14;
```

Stylelint (17.10.0)

[error] Unexpected empty line before $-variable (scss/dollar-variable-empty-line-before)

(scss/dollar-variable-empty-line-before)